vncviewer/OptionsDialog.h

```diff
@@ -22,6 +22,8 @@
 #include <map>
 
 #include <FL/Fl_Window.H>
+#include "Monitors.h"
+#include "MonitorArrangement.h"
```

Comment:
These are not necessary and should be avoided. Look at how we handle the FLTK widget classes.